zenodo/modules/theme/static/scss/record.scss

```
@@ -305,3 +305,27 @@
     font-size: 36px;
   }
 }
+
+.record-description {
+  table {
+    margin: 1rem 0;
+  }
+
+  th, td {
+    border-style: solid;
+    border-width: thin;
+    padding: 0.5rem 2rem;
+  }
+}
+
+.record-notes {
+  table {
+    margin: 1rem 0;
+  }
+
+  th, td {
+    border-style: solid;
+    border-width: thin;
+    padding: 0.5rem 2rem;
+  }
+}
```

```suggestion
.record-description, .record-notes {
  table {
    margin: 1rem 0;
  }

  th, td {
    border-style: solid;
    border-width: thin;
    padding: 0.5rem 2rem;
  }
}
```
For avoiding duplication